flink-runtime/src/main/java/org/apache/flink/runtime/jobmaster/JobMaster.java

```diff
@@ -303,7 +308,8 @@ public void onUnknownDeploymentsOf(
         this.highAvailabilityServices = checkNotNull(highAvailabilityService);
         this.blobWriter = jobManagerSharedServices.getBlobWriter();
         this.futureExecutor = jobManagerSharedServices.getFutureExecutor();
```

Comment:
Is there a reason we dont wrap this one as well?

Reply:
No, I just overlooked it :)